tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -3142,8 +3223,16 @@ HRESULT CorProfiler::GenerateVoidILStartupMethod(const ModuleID module_id, mdMet
     pNewInstr->m_Arg32 = 0;
     rewriter_void.InsertBefore(pFirstInstr, pNewInstr);
 
-    // Set the false branch target
-    pBranchFalseInstr->m_pTarget = pNewInstr;
+    if (pBranchTrueInstr != nullptr)
```

Comment:
small naming nit: Can we rename the branch instructions to something like this?
- `pBranchTrueInstr` => `pIsFullyTrustedBranch`
- `pBranchFalseInstr` => `pIsNotAlreadyLoadedBranch`

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/4108/commits/2edadb0a98e829cf473fb79d6840515e547a3458